docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -0,0 +1,263 @@
+# Migrating to v6
+
+<p class="description">This guide explains how and why to migrate from Material UI v5 to v6.</p>
+
+## Start using the alpha release
+
+In the `package.json` file, change the package version from `latest` to `next`.
+
+```diff title="package.json"
+-"@mui/material": "latest",
++"@mui/material": "next",
+```
+
+Optionally, if you are using one of these packages, be sure to change the version to `next` too:
+
+- `@mui/icons-material`
+- `@mui/system`
+- `@mui/lab`
+- `@mui/material-nextjs`
+- `@mui/styled-engine-sc`
+- `@mui/utils`
+
+Using `next` ensures your project always uses the latest v6 alpha release.
+Alternatively, you can also target and fix it to a specific version, for example, `6.0.0-alpha.0`.
+
+## Why you should migrate
+
+Material UI v6's biggest highlight is the inclusion of Pigment CSS, a next-gen zero-runtime CSS-in-JS library, as an opt-in styling engine.
+Using it will make your project compatible with React Server Components and reduce its bundle size due to the styles extraction, avoiding client-side recalculation.
+
+Aside from that, the CSS variables API for Material UI is now stable in v6.
+Relying on CSS variables allows for more intricate customization possibilities and better overall DX.
+
+A major feedback we've learned from the v5 cycle is to reduce the number of breaking changes to the smallest.
+Material UI v6 does exactly that: minimal breaking changes, including only browser support updates and a Node.js version bump.
+These updates can, for the most part, be migrated automatically via codemods.
+
+Last but not least, Material UI v6 also includes new features, such as container queries and a theme utility for styling different color schemes.
+
+## Supported browsers and Node versions
+
+The targets of the default bundle have changed in v6.
+
+The exact versions will be pinned on release from the browserslist query `"> 0.5%, last 2 versions, Firefox ESR, not dead, safari >= 15.4, iOS >= 15.4"`.
+
+<!-- #stable-snapshot -->
+
+- Node.js 18 (up from 12)
+- Chrome 109 (up from 90)
+- Edge 121 (up from 91)
+- Firefox 115 (up from 78)
+- Safari 15.4 in both macOS and iOS (up from 14 in macOS and 12.5 in iOS)
+- and more (see [.browserslistrc (`stable` entry)](https://github.com/mui/material-ui/blob/v6.0.0/.browserslistrc#L11))
+
+### Removed support for IE 11
+
+Support for IE 11 is completely removed, by dropping the legacy bundle and all IE 11 related code.
+This allows us to decrease bundle size and keep the scope manageable.
+If you need to support IE 11, you can use Material UI v5's [legacy bundle](https://v5.mui.com/material-ui/guides/minimizing-bundle-size/#legacy-bundle), but it won't get updates or bug fixes.
+
+## Update React & TypeScript version
+
+### Update React
+
+The minimum supported version of React is v17.0.0 (the same as v5).
```

Comment:
Let's add the command to update React here for added convenience

Reply:
Added.